lib/reporting/reports/suppliers/base.rb

```diff
@@ -40,10 +40,10 @@ def columns
             product:,
             variant_unit_name:,
             quantity:,
-            total_excl_vat_and_fees:,
+            total_excl_fees_and_tax:,
             total_excl_vat:,
```

Comment:
It's the existing locale and being used in many reports that's why it's being kept the same. We can create a small refactor tech debt to fix this across the app.

Reply:
Sorry, I was just looking at the variable and method names.
I see what you mean now: there's an existing  [`total_excl_vat`](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/locales/en.yml#L3325-L3326), but we're adding keys like `total_excl_fees_and_tax`. Oh well, we are hopefully smart enough to figure it out ;)

I will just ask for another opinion in Slack to make sure we are heading in the right direction.

Reply:
Ah yes, exactly. Sorry I skipped this point before, I used this existing locale for reusability purposes and avoided duplication. But yes, I think we can afford this type of duplication 😬

Reply:
Ah, yes. Sorry I didn't mention before I used the existing locale for reusability purposes and tried to avoid duplication. But yes, you are right we can afford this type of duplication for improvement purposes.

Reply:
No worries. We have one agreement in [#delivery-circle](https://openfoodnetwork.slack.com/archives/C01T75H6G0Z/p1731552153716269?thread_ts=1731552139.728729&cid=C01T75H6G0Z), which I think is enough to proceed ⏩